operator/src/test/java/oracle/kubernetes/operator/steps/ManagedServerUpIteratorStepTest.java

```diff
@@ -156,34 +171,44 @@ private static V1ObjectMeta withNames(V1ObjectMeta objectMeta, String serverName
             .putLabelsItem(SERVERNAME_LABEL, serverName);
   }
 
+  protected TestUtils.ConsoleHandlerMemento configureOperatorLogger() {
```

Comment:
I can understand a desire to avoid line breaks, but in this case, I think it is clearer to leave this call to `silenceOperatorLogger` inline, with `ignoringLoggedExceptions` on the next line.

Reply:
Thanks for the review comment, I have made the change as per your suggestion.